### ISSUE TITLE
Image caption button doesn't place focus into editable after execution

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#12165]
## 📝 Provide detailed reproduction steps (if any)

1. Open sample in https://codepen.io/mlewand/pen/VwXMdrL (uses decoupled editor with image caption)
2. Focus the editable area.
3. Select image on the right hand side.
4. Press alt+f10 to move the focus to the toolbar.
5. Press space to execute image caption command.

### ✔️ Expected result

The selection should be moved to editable just like in bold, underline, blockquote or any other command that modifies the content.

### ❌ Actual result

Focus remains in the toolbar which is inconsistent with other UI.

## 📃 Other details

* Browser: …
* OS: …
* First affected CKEditor version: …
* Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
